README.md

```diff
@@ -1,3 +1,517 @@
+        pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+        pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+      end
+      ```
+  4. Run `pod install` inside the same folder where the `pod` file was created
+  5. Open the `.xcworkspace` file that was created and run
+
+#### Android
+
+1. Add the following lines to `android/settings.gradle`:
+   
+   ```gradle
+   include ':react-native-datetimepicker'
+   project(':react-native-datetimepicker').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-datetimepicker/android')
+   ```
+
+2. Add into `android/gradle.properties`:
+   
+   ```gradle
+   android.useAndroidX=true
+   android.enableJetifier=true
+   ```
+
+3. Add the compile line to the dependencies in `android/app/build.gradle`:
+
+   ```gradle
+   dependencies {
+       ...
+       implementation project(':react-native-datetimepicker')
+   }
+   ```
+
+4. Add the import and link the package in `MainApplication.java`:
+
+   ```java
+   import com.reactcommunity.rndatetimepicker.RNDateTimePickerPackage; // <-- add this import
+
+   public class MainApplication extends Application implements ReactApplication {
+       @Override
+       protected List<ReactPackage> getPackages() {
+           return Arrays.<ReactPackage>asList(
+               new MainReactPackage(),
+               new RNDateTimePickerPackage() // <-- add this line
+           );
+       }
+   }
+   ```
+
+## Tests
+
+### Jest
+
+```
+$ npm install
+$ npm run test
+```
+
+### Detox
+
+Detox is a gray box end-to-end testing and automation library for mobile apps.
+- [Dependencies required](https://github.com/wix/Detox/blob/master/docs/Introduction.GettingStarted.md#step-1-install-dependencies)
+
+For cleaning all the detox builds just run `npm run detox:clean`.
+
+#### iOS
+
+- debug:
+  ```sh
+  npm install
+  # Debug requires to run Metro Bundler
+  npm run start
+  npm run detox:ios:build:debug
+  npm run detox:ios:test:debug
+  ```
+
+- release:
+  
+  ```sh
+  npm install
+  npm run detox:ios:build:release
+  npm run detox:ios:test:release
+  ```
+
+#### Android
+
+  An existing Android emulator is required to match the name defined in `detox.configurations.android.emu.debug.name` and `detox.configurations.android.emu.release.name` inside the `package.json`.
+
+  - debug:
+    ```sh
+    npm install
+    # Debug requires to run Metro Bundler
+    npm run start
+    npm run detox:android:build:debug
+    npm run detox:android:test:debug
+    ```
+
+  - release:
+    
+    ```sh
+    npm install
+    npm run detox:android:build:release
+    npm run detox:android:test:release
+    ```
+
+## Running the example app
+
+1. Run `npm install`
+2. Run `npm run start` for starting Metro Bundler
+3. Run `npm run start:ios` or `npm run start:android`
+
+## General Usage
```

Comment:
This should more on top, above all the `contributing to the component`  sections